src/scraper.py

```diff
@@ -57,40 +103,147 @@ def load_code_systems():
+        while len(response.codeSystemConcepts) == page_size:
+            page_number += 1
+            response = request_without_timeout(
+                30,
+                service.getCodeSystemConceptsByCodeSystemOid,
+                code_system_oid,
+                page_number,
+                page_size,
+            )
+            code_system_concepts += response.codeSystemConcepts
+            time.sleep(5)
+
+        # Create a table for code system concepts
+        if not table_created:
+            code_system_concept_example = code_system_concepts[0].__dict__
+            print(code_system_concept_example)
+            create_table_sql = generate_create_table_sql(
+                "code_system_concept", code_system_concept_example
+            )
+            print(create_table_sql)
+            cur.execute(create_table_sql)
+            table_created = True
+            # Drop all rows from the table
+            cur.execute("DELETE FROM code_system_concept;")
+
+        # Insert code system concepts into the table
+        for code_system_concept in code_system_concepts:
+            code_system_concept_dict = code_system_concept.__dict__
+            columns = code_system_concept_dict.keys()
+            values = code_system_concept_dict.values()
+            # Join column names and values with commas
+            columns = ", ".join(columns)
+            values = join_values(values)
+            insert_sql = sql.SQL(
+                "INSERT INTO code_system_concept ({}) VALUES ({});"
+            ).format(sql.SQL(columns), sql.SQL(values))
+            cur.execute(insert_sql)
+
+        print(f"Inserted {len(code_system_concepts)
+                          } code system concepts for code system {code_system_oid}")
+        conn.commit()
+        time.sleep(5)
+
+
+def check_code_system_concept_count():
+    # Get counts from the database
+    cur.execute(
+        "SELECT codesystemoid, COUNT(*) FROM code_system_concept GROUP BY codesystemoid;"
+    )
+    counts_in_db = dict(cur.fetchall())
+
+    counts_from_api = {}
+    mismatched_counts = []
+
+    for code_system_oid, count in counts_in_db.items():
+        response = request_without_timeout(
+            15, service.getCodeSystemConceptsByCodeSystemOid, code_system_oid, 1, 1
+        )
+        expected_count = response.totalResults
+        counts_from_api[code_system_oid] = expected_count
+        if expected_count != count:
+            print("MISMATCH!!!")
+            print(
+                f"Expected count for code system {code_system_oid}: {expected_count}, actual count: {count}"
+            )
+            mismatched_counts.append(code_system_oid)
+
+    if len(mismatched_counts) == 0:
+        print("All counts match")
+    else:
+        print("Mismatched counts: ", mismatched_counts)
+
+
 # Connect to the database
 conn = psycopg2.connect("dbname=phinvads user=postgres password=cdc host=db")
 cur = conn.cursor()
```

Comment:
Possible nitpick: is there an advantage to having these variables declared way down here instead of at the top of the scraper? I found myself hunting them out since they're called in previous functions.

Reply:
Good call! Totally makes sense to have these at the top